clients/redshift/cast.go

```diff
@@ -5,25 +5,24 @@ import (
 
 	"github.com/artie-labs/transfer/lib/config/constants"
 	"github.com/artie-labs/transfer/lib/typing"
-	"github.com/artie-labs/transfer/lib/typing/columns"
 	"github.com/artie-labs/transfer/lib/typing/values"
 )
 
 const maxRedshiftLength int32 = 65535
 
 // replaceExceededValues - takes `colVal` any and `colKind` columns.Column and replaces the value with an empty string if it exceeds the max length.
```

Comment:
Oh this one also has the wrong type in it

Reply:
Wow the comment for this one is verbose. Removing